src/test/java/seedu/address/testutil/TypicalNote.java

```diff
@@ -9,7 +9,7 @@
     public static final Note NOTE_DONE = new NoteBuilder()
```

Comment:
might want to change the name to be less confusing, since this note is also not done yet